README.md

```diff
@@ -158,9 +166,599 @@ search.start();
+- ✅ initialUiState: Determine the search state on app start.
+- ✅ onStateChange: Change search state on change (see option above).
+- ✅ stalledSearchDelay: Time in ms before search is considered stalled. [Used for loader](https://www.algolia.com/doc/guides/building-search-ui/going-further/improve-performance/js/#using-the-searchbox).
+- ✅ routing: browser URL synchronization, search parameters appear in current URL ([guide](https://www.algolia.com/doc/guides/building-search-ui/going-further/routing-urls/js/)).
+- ✅ insightsClient: Hook analytics to search actions ([see insight section](#insight)).
+
+```js
+const search = instantsearch({
+  indexName: 'instant_search',
+  searchClient: instantMeiliSearch(
+    'https://demos.meilisearch.com',
+    'dc3fedaf922de8937fdea01f0a7d59557f1fd31832cb8440ce94231cfdde7f25',
+    {...instantMeiliSearchOptions}
+  ),
+  {...instantSearchOptions}
+})
+```
+
+### ❌ Index
+
+[Index references](https://www.algolia.com/doc/api-reference/widgets/index-widget/js/)
+
+`Index` is the component that lets you apply widgets to a dedicated index. It’s useful if you want to build an interface that targets multiple indices.
+
+Not compatible as MeiliSearch does not support federated search on multiple indexes.
+
+If you'd like to see federated search implemented please vote for it in the [roadmap](https://roadmap.meilisearch.com/c/74-multi-index-search?utm_medium=social&utm_source=portal_share).
+
+### ✅ SearchBox
+
+[SearchBox references](https://www.algolia.com/doc/api-reference/widgets/search-box/js/)
+
+The `searchBox` widget is used to let the user perform a text-based query.
+
+- ✅ container: The CSS Selector or HTMLElement to insert the widget into. _required_
+- ✅ placeholder: Placeholder of the search box.
+- ✅ autofocus: Wether the search box is focused on arrival.
+- ✅ searchAsYouType: Whether result appear as you type or after pressing enter.
+- ❌ showReset: Did not work with both algoliaSearch and instantMeiliSearche
+- ❌ showSubmit: Did not work with both algoliaSearch and instantMeiliSearch
+- ✅ showLoadingIndicator: Whether to show the spinning loader.
+- ✅ queryHook: A function that is called just before the search is triggered.
+- ✅ templates: The templates to use for the widget.
+- ✅ cssClasses: The CSS classes to override.
+
+```js
+instantsearch.widgets.searchBox({
+  container: '#searchbox',
+  autofocus: true,
+  ...searchBoxOptions
+}),
+```
+
+### ✅ Configure
+
+- [Configure references](https://www.algolia.com/doc/api-reference/widgets/configure/js/): See #389
+
+The `configure` widget lets you provide raw search parameters to the Algolia API without rendering anything.
+
+Because these are the search parameters of AlgoliaSearch and not the InstantSearch parameters, some of them are ignored by InstantSearch.
+Since we do not act as AlgoliaSearch on search parameters detailed compatibility can be found in [this issue](https://github.com/meilisearch/instant-meilisearch/issues/389).
```

Comment:
```suggestion
Since we do not act as AlgoliaSearch on search parameters, detailed compatibility can be found in [this issue](https://github.com/meilisearch/instant-meilisearch/issues/389).<br>
```